app/Client/GuzzlePublisherBannerClassifier.php

```diff
@@ -41,9 +41,9 @@ public function __construct(Client $client)
         $this->client = $client;
     }
 
-    public function fetchBannersClassification(array $bannerIds): Collection
+    public function fetchBannersClassification(array $bannerUuids): Collection
     {
-        $body = json_encode($bannerIds);
+        $body = json_encode($bannerUuids);
```

Comment:
This should be pointed out earlier. This function uses variable $body in two different contexts which could be misleading. Please rename second assignment to $rseponseBody.